### DISCUSSION TITLE
Can I pass string in the ports?

### DISCUSSION THREAD

**Discussion by sheershak-agarwal** [nasa/fprime#1039] [Q&A] [answered]
Can I pass strings of fixed length in the ports?

**Comment by Joshua-Anderson** [comment, marked as answer]
Yep, just add an argument of type string and the fixed length. [Example from FileDownlink](https://github.com/nasa/fprime/blob/devel/Svc/FileDownlink/SendFileRequestPortAi.xml#L8)

**Reply by aggelosmots** [reply]
The Example from FileDownlink is not opening for me? Is there any other example?

**Reply by LeStarch** [reply]
https://github.com/nasa/fprime/blob/91c578ee37c01ba92e556dbca0287d82f9f18b95/Svc/FileDownlinkPorts/FileDownlinkPorts.fpp#L24 

We've updated the modeling language.
